this_is_the_way.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def this_is_the_way():
+    res_lst = []
```

Comment:
dont use abbreviations in code - result_lsit and not res_list - you want the programmer reading your code to understand it as much a possible